packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -93,15 +94,18 @@ export default class Editor {
             this.contenteditableChanged = true;
         }
 
-        // 8. Disable these operations for firefox since its behavior is usually wrong
+        // 8. Make settings change to make editor work better
         // Catch any possible exception since this should not block the initialization of editor
         try {
+            // Disable these operations for firefox since its behavior is usually wrong
             this.core.document.execCommand(DocumentCommand.EnableObjectResizing, false, <string>(
                 (<any>false)
             ));
             this.core.document.execCommand(DocumentCommand.EnableInlineTableEditing, false, <
                 string
             >(<any>false));
+            // Change the default paragraph separater to DIV. This is mainly for IE since its default setting is P
+            this.core.document.execCommand(DocumentCommand.DefaultParagraphSeparator, false, 'div');

```

Comment:
If the purpose of these operations is for Firefox and IE, why isn't there a browser check?